e2e/portalicious/tests/ManageTeam/ScopeVisibleOnlyWhenSetToTrue.spec.ts

```diff
@@ -0,0 +1,42 @@
+import BasePage from '@121-e2e/portalicious/pages/BasePage';
+import LoginPage from '@121-e2e/portalicious/pages/LoginPage';
+import ProjectTeam from '@121-e2e/portalicious/pages/ProjectTeam';
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import { resetDB } from '@121-service/test/helpers/utility.helper';
+import { test } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await resetDB(SeedScript.nlrcMultiple);
+
+  // Login
+  const loginPage = new LoginPage(page);
+  await page.goto('/');
+  await loginPage.login(
+    process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN,
+    process.env.USERCONFIG_121_SERVICE_PASSWORD_ADMIN,
+  );
+});
+
+test('[29761] "Scope" column should only be loaded when "enableScope": is set to true under the program configuration', async ({
+  page,
+}) => {
+  const basePage = new BasePage(page);
+  const manageTeam = new ProjectTeam(page);
+  const projectTitle1 = 'NLRC Direct Digital Aid Program (PV)';
+  const projectTitle2 = 'NLRC OCW program';
```

Comment:
Do we need to hard-code these dynamic texts here?

Wouldn't "go to project-team-page; check for column" as a test-scenario be enough?

Reply:
I could just change it to navigate to correct urls and then no text is needed.

Reply:
Done